### ISSUE TITLE
button-group rely on whitespace for inbetween spacing

### ISSUE THREAD

**Issue by stepandersen** [PayEx/design.payex.com#359]
Buttons in button-group (or just two sibling buttons really) are rendered with spacing inbetween because there is a white-space character there. (In code that white-space is actually a line-break and maybe a couple of tabs.)

Minifiers remove this spacing and the buttons are rendered like:
![image](https://user-images.githubusercontent.com/2443474/57452460-1ec47b80-7264-11e9-9842-55183fedf7f1.png)

Some potential solutions:
https://css-tricks.com/fighting-the-space-between-inline-block-elements/

**Comment by Akswii** [comment]
So intended behavior should be no white-space and fix "air" between the buttons with margin?

**Comment by stepandersen** [comment]
Yes, the buttons should have the same spacing inbetween regardless of white-spaces.
I suppose margin-right and margin-left makes the most sense, even though DesignGuide doesn't recommend right aligned buttons.

Something like this perhaps:
```
.button-group {
  display: flex;
}
.button-group .btn {
  margin-left: 2px;
  margin-right: 2px;
}
.button-group .btn:first-child {
  margin-left: 0;
}
.button-group .btn:last-child {
  margin-right: 0;
}
```

**Comment by Akswii** [comment]
We see the need to support compiled templates, and like your solution. But need to add flex-wrap:wrap and consider if we are going to solve this issue for buttons not wrapped in a .button-group.
